packages/orbit-components/src/Skeleton/__tests__/index.test.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import * as React from "react";
+import { render, screen } from "@testing-library/react";
+
+import Skeleton from "..";
+
+const UID = ({ children }) => children("154");
+
+jest.mock("react-uid", () => ({
+  UID,
+}));
+
+describe("Skeleton", () => {
+  it("it should expected DOM output", () => {
+    const dataTest = "test";
+    const children = "kek";
+    const ariaLabelledby = "bur";
+
+    render(
+      <Skeleton dataTest={dataTest} title="Loading" ariaLabelledby={ariaLabelledby}>
+        <rect>{children}</rect>
+      </Skeleton>,
+    );
+
+    expect(screen.getByTestId(dataTest)).toBeInTheDocument();
```

Comment:
In the future we can switch to simply calling `getBy*` without `expect` because `getBy*` itself fails when it doesn't exist, so it never reaches the `expect` functionality.